tests/tests/lightspeed-install.cy.ts

```
@@ -1059,4 +1160,225 @@ metadata:
+      const SECOND_PROMPT = 'Show me another dashboard';
+      const SECOND_TOOL_NAME = 'metrics';
+      const SECOND_URI = 'mcp://test-server/resources/metrics';
+
+      const SECOND_HTML = `<!DOCTYPE html>
+<html lang="en">
+<head><title>Metrics</title></head>
+<body><div class="metrics">Metrics Dashboard</div></body>
+</html>`;
+
+      cy.interceptMCPQuery('mcpQuery1', MCP_PROMPT, MCP_TOOL_NAME, MCP_UI_RESOURCE_URI);
+      cy.interceptMCPResources('mcpResources1', SAMPLE_MCP_HTML, 'test-server', MCP_UI_RESOURCE_URI);
```

ESLint

[error] Replace `'mcpResources1',·SAMPLE_MCP_HTML,·'test-server',·MCP_UI_RESOURCE_URI` with `⏎········'mcpResources1',⏎········SAMPLE_MCP_HTML,⏎········'test-server',⏎········MCP_UI_RESOURCE_URI,⏎······`

(prettier/prettier)